cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,247 +3,378 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
+@margin__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@spine-width__px: 5px;
+@spine-color: @gray-40;
+@node-width__desktop__px: 405px;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dca185a0d8c0205efa146b0695e88fa92ea3df97